From 58ef0c40b6fc098f9a02f816fc0470177a0d8f70 Mon Sep 17 00:00:00 2001
From: Fushan Wen <qydwhotmail@gmail.com>
Date: Sat, 24 Feb 2024 08:57:20 +0800
Subject: [PATCH] components/dialogs: fall back to null target when
 dialogButtonBox is null

undefined is not a valid value for the target property.

Amends 5fe7ad2ed05bf7521411fe81f724a1e730cd5008


(cherry picked from commit b4214478be857cfc7044c1df599b7da1124c7a6a)
---
 components/dialogs/SystemDialog.qml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/dialogs/SystemDialog.qml b/components/dialogs/SystemDialog.qml
index 25c194e891f..8c19fa33d33 100644
--- a/components/dialogs/SystemDialog.qml
+++ b/components/dialogs/SystemDialog.qml
@@ -100,7 +100,7 @@ Kirigami.AbstractApplicationWindow {
     }
 
     Binding {
-        target: dialogButtonBox?.standardButton(QQC2.DialogButtonBox.Ok)
+        target: dialogButtonBox?.standardButton(QQC2.DialogButtonBox.Ok) ?? null
         property: "enabled"
         when: dialogButtonBox?.standardButtons & QQC2.DialogButtonBox.Ok
         value: root.acceptable
-- 
GitLab

